core/eds.go

```diff
@@ -21,19 +20,25 @@ import (
 	"github.com/celestiaorg/celestia-node/store"
 )
 
+// isEmptyBlockRef returns true if the application considers the given block data
+// empty at a given version.
+func isEmptyBlockRef(data *coretypes.Data) bool {
+	return len(data.Txs) == 0
+}
```

Comment:
this function was removed from celestia-app, it's small so I just duplicated it here.